Formula/pango.rb

```diff
@@ -11,6 +11,8 @@ class Pango < Formula
     sha256 "1e9be870617caba7603eb1b8953eed684a863035ae2becabe0dd86b9f96a540a" => :mavericks
   end
 
+  patch :p1, :DATA
```

Comment:
Can you report this upstream & include a link to the upstream bug report above this line? Thanks!